packages/circuit-ui/components/Select/Select.tsx

```diff
@@ -361,11 +363,7 @@ export const Select = forwardRef(
             {...props}
             onChange={handleChange}
           >
-            {!value && !defaultValue && (
-              <option key="placeholder" value="">
-                {placeholder}
-              </option>
-            )}
+            {!value && !defaultValue && <option value="">{placeholder}</option>}
```

Comment:
The `key` prop is required by React to keep track of the elements.

Reply:
There's a mapping below for the other options. React just sees a `select` element with a bunch of `children`. 

This was specifically added because it was causing issues.

Reply:
Aha, the classic. So weird! This isn't an iterator. So does it mean that this would also need a key?

```jsx
const divs = [1, 2];
const Component () => (
  <div>
    <div>I'm div 0!</div>
    {divs.map(div => (
      <div key={div}>I'm div {div}!</div>
    )}
  </div>
);
```

Reply:
I think so 🤔

Reply:
Last iteration that avoids using the index as a unique key: [b7acd7d](https://github.com/sumup-oss/circuit-ui/pull/1528/commits/b7acd7d7f20e0f429f0619b95a7a91e25b95d750)